### ISSUE TITLE
Changing synth away from OneCore crashes NVDA

### ISSUE THREAD

**Issue by timothytylee** [nvaccess/nvda#8648]
### Steps to reproduce:
1.  Install portable copy of NVDA on Windows 10.
2.  Start portable NVDA and exit, so that OneCore is saved as chosen synth.
3.  Start NVDA again and open Settings dialog box
4.  Choose a different synthesizer, such as eSpeak NG, then click OK.

### Actual behavior:
NVDA crashes, and starts with OneCore synth again.

### Expected behavior:
NVDA should switch to the newly selected synthesizer.

### System configuration:
VirtualBox VM with 4 virtual cores.

#### NVDA Installed/portable/running from source:
Portable NVDA.

#### NVDA version:
2018.2.1

#### Windows version:
Windows 10, 1803

#### Name and version of other software in use when reproducing the issue:
No other programs running.

#### Does the issue still occur after restarting your PC?
The problem is not fixed by reboot.

#### Have you tried any other versions of NVDA?
Tried running NVDA from source, using master branch in git.  Same problem occurs.

**Comment by timothytylee** [comment]
[nvda-change-synth-crash.log](https://github.com/nvaccess/nvda/files/2300088/nvda-change-synth-crash.log)

**Comment by timothytylee** [comment]
Attached is a log of the crash.

**Comment by Adriani90** [comment]
@timothytylee I cannot reproduce this issue in NVDA 2019.1.1. Are you still able to reproduce it?

**Comment by 3nuc** [comment]
I can reproduce it on 2019.2.1. I'm not running NVDA in a virutal machine though. I'm on normal Windows 10.

**Comment by Adriani90** [comment]
@3nuc could you please try on NVDA 2019.3 Beta 1? Which system language are you using?

**Comment by 3nuc** [comment]
@Adriani90 Tried 2019.3 Beta 1. The issue is present on this version also.
I'm not sure if it actually crashes NVDA. NVDA restarts after it closes in this case

Here's my language settings:
![image](https://user-images.githubusercontent.com/5359825/71345436-a4800780-2565-11ea-9e8a-27545cd94e44.png)
My Windows UI is in English but preferred language for apps is Polish.

**Comment by Adriani90** [comment]
This coul be related or even duplicate of #10607. cc: @lukaszgo1, @Qchristensen, @michaelDCurran

**Comment by 3nuc** [comment]
My nvda path is "C:\Program Files (x86)\NVDA\nvda.exe".

I read the issue you posted and got the idea of also linking to my logs with debug loglevel like the OP of that issue. I started NVDA from the console with the appropriate commands (-l 10 -f log.txt) as administrator only to realize I can no longer reproduce the issue.

I'm on 2019.3 beta 1 (checked in about page). Even now if I launch from start menu I can switch to espeak ng and back to windows mobile sounds all the time and NVDA doesn't crash.

My only guess is that a single running of nvda as admin could've solved the issue? I have no other guesses really

**Comment by Adriani90** [comment]
@3nuc thanks for reporting. Good to hear that your issue is not occuring anymore. I am closing this as works for me. If the issue still occurs, please feel free to comment on this issue and we can reopen it.
